.github/workflows/release-image.yml

```diff
@@ -0,0 +1,28 @@
+name: release image
+on:
+  push:
+    tags:
+      - 'image-v*' # push events for tags matching image-v for version (image-v1.0, etc)
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: set env
+        run: echo ::set-env name=RELEASE_VERSION::$(echo ${GITHUB_REF:17}) # refs/tags/image-v1.0.0 substring starting at 1.0.0
+      - name: setup buildx
+        uses: docker/setup-buildx-action@v1
+      - name: login to GitHub container registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
+      - name: build and push
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          file: 'vmss-prototype/Dockerfile'
+          tags: |
+            ghcr.io/owner/templated-repo:${{ env.RELEASE_VERSION }}

```

Comment:
Does this replace owner/templated-repo somehow or is that just placeholder for us to put the right thing?

Reply:
Looking on the web:  https://github.com/docker/build-push-action

Reply:
You found the bug, good job :)